From 112e059839b681dbbc0260c59a0d453de7206ff5 Mon Sep 17 00:00:00 2001
From: Antonio Terceiro <terceiro@debian.org>
Date: Thu, 16 Jul 2015 12:11:05 -0300
Subject: [PATCH] lxc-debian: skip security updates for unstable/sid

There is no such thing as security support for unstable/sid.
---
 templates/lxc-debian.in | 5 +++++
 1 file changed, 5 insertions(+)

--- a/templates/lxc-debian.in
+++ b/templates/lxc-debian.in
@@ -173,8 +173,13 @@ write_sourceslist()
 
     cat >> "${rootfs}/etc/apt/sources.list" << EOF
 ${prefix} $MIRROR          ${release}         main contrib non-free
+EOF
+
+    if [ "$release" != "unstable" -a "$release" != "sid" ]; then
+      cat >> "${rootfs}/etc/apt/sources.list" << EOF
 ${prefix} $SECURITY_MIRROR ${release}/updates main contrib non-free
 EOF
+    fi
 }
 
 configure_debian_systemd()
